src/components/ContributionForm/ContributionForm.jsx

```diff
@@ -0,0 +1,32 @@
+import { useState, useEffect } from "react";
+
+function ContributionForm() {
+  const [amount, setAmount] = useState("");
+  const [campaignId, setCampaignId] = useState("");
+
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+
+    const contributionData = {
```

Comment:
there isn't really any need for this, since you don't use these values at any point